modules/avm/variables.tf

```
@@ -4,6 +4,30 @@ variable "account_name" {
   description = "Name of the AWS Service Catalog provisioned account (overrides computed name from the `name` variable)"
 }
 
+variable "account_password_policy" {
+  type = object({
+    allow_users_to_change        = bool
+    max_age                      = number
+    minimum_length               = number
+    require_lowercase_characters = bool
+    require_numbers              = bool
+    require_symbols              = bool
+    require_uppercase_characters = bool
+    reuse_prevention_history     = number
+  })
+  default = {
+    allow_users_to_change        = true
+    max_age                      = 90
+    minimum_length               = 14
+    require_lowercase_characters = true
+    require_numbers              = true
+    require_symbols              = true
+    require_uppercase_characters = true
+    reuse_prevention_history     = 24
+  }
+  description = "The password policy parameters to set for the AWS account"
```

```suggestion
  description = "AWS account password policy parameters"
```

Better to keep the descriptions terse when possible.

---

agree